src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -14788,6 +14788,210 @@ IEnumerable<string> M() => [string.Empty, .. ($$
+            namespace PrimaryConstructor;
+
+            public {{typeKind}} Point(int X, int Y)
+            {
+                public static Point Parse(string line)
+                {
+                    $$
+                }
+            }
+            """;
+        await VerifyExpectedItemsAsync(markup, [
+            ItemExpectation.Absent("X"),
+            ItemExpectation.Absent("Y"),
+        ]);
+    }
+
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/74327")]
+    [InlineData("class")]
+    [InlineData("record class")]
+    public async Task RecommendedPrimaryConstructorParameters02(string typeKind)
+    {
+        var markup = $$"""
+            namespace PrimaryConstructor;
+
+            public abstract {{typeKind}} BasePoint(int X);
+
+            public {{typeKind}} Point(int X, int Y)
+                : BasePoint(X)
+            {
+                public static Point Parse(string line)
+                {
+                    $$
+                }
+            }
+            """;
+        await VerifyExpectedItemsAsync(markup, [
+            ItemExpectation.Absent("X"),
+            ItemExpectation.Absent("Y"),
+        ]);
+    }
+
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/74327")]
+    [InlineData("class")]
+    [InlineData("record class")]
+    public async Task RecommendedPrimaryConstructorParameters03(string typeKind)
+    {
+        var markup = $$"""
+            namespace PrimaryConstructor;
+
+            public abstract {{typeKind}} BasePoint(int X);
+
+            public {{typeKind}} Point(int X, int Y)
+                : BasePoint(X)
+            {
+                public int Y { get; init; } = Y;
+
+                public static Point Parse(string line)
+                {
+                    $$
+                }
+            }
+            """;
+        await VerifyExpectedItemsAsync(markup, [
+            ItemExpectation.Absent("X"),
+            ItemExpectation.Absent("Y"),
+        ]);
+    }
+
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/74327")]
+    [InlineData("class")]
+    [InlineData("struct")]
+    [InlineData("record class")]
+    [InlineData("record struct")]
+    public async Task RecommendedPrimaryConstructorParameters04(string typeKind)
+    {
+        var markup = $$"""
+            namespace PrimaryConstructor;
+
+            public {{typeKind}} Point(int X, int Y)
+            {
+                public static Point Parse(string line)
+                {
+                    var n = nameof($$
+                }
+            }
+            """;
+        await VerifyExpectedItemsAsync(markup, [
+            ItemExpectation.Exists("X"),
+            ItemExpectation.Exists("Y"),
+        ]);
+    }
+
+    [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/74327")]
+    [InlineData("record class")]
+    [InlineData("class", Skip = "Class primary constructor parameter used in base constructor not recommended even inside nameof")]
```

Comment:
Don't add skipped tests. Just have the tests demonstrate what the behavior is currently

Reply:
Also, this seems trivially fixable

Reply:
I'd prefer fixing this in another PR. My intention with adding the skipped test is to demonstrate an existing bug. Otherwise it would be seen as intended behavior.

Reply:
Unskip and leave a comment. Skipped tests serve no value. We only want to skip for flakey issues we're unblocking ci for.